src/main/java/ua/lviv/iot/managers/ConcertHallManager.java

```diff
@@ -0,0 +1,61 @@
+package ua.lviv.iot.managers;
+
+
+import lombok.Data;
+import lombok.NonNull;
+import ua.lviv.iot.models.Performance;
+import ua.lviv.iot.models.SortOrder;
+
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.LinkedList;
+
+@Data
+@NonNull

```

Comment:
What does this @NonNull belong to? Do you understand which methods would receive non-null check when you annotate the whole class?

Reply:
Fixed, only fields are NotNull now